City-of-Helsinki/drupal-hdbt
# UHF-8215: News list params

# [UHF-8215](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8215)
This branch:
Adds tags to news list component

Platform config PR: 
Adds query string to the news archive link at the bottom of the news list

## What was done

This PR:
* Moved news tags list to its own template for reusability
* Added tags to news-list component
* Added block for tags display to the component template

Platform config PR:
* Added logic for passing query params to news archive

## How to install

* Start up an instance of your choice
* Get the correct branches:
  * `composer require drupal/hdbt:dev-UHF-8215-news-list-params drupal/helfi_platform_config:dev-UHF-8215-news-list-params`
* Run `make drush-cr`

## How to test
* Add or edit a landing page to include paragraph of type "News list"
* Select some taxonomy terms to go along with it from the paragraph options
* View the page. Check for these:
  * Selected taxonomy terms should be displayed as tag pills on the top of the news list paragraph
  * The button link at the bottom of the paragraph should take you to news archive with the selected terms as filters

## Designers review



## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/compare/UHF-8215-news-list-params?expand=1


[UHF-8215]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8215?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
